CRM/Civicase/Upgrader.php

```diff
@@ -65,8 +65,8 @@ public function install() {
         'option_group_id' => 'activity_type',
         'name' => ['IN' => $activityTypes],
         'api.OptionValue.setvalue' => [
-          'field' => 'grouping',
-          'value' => $grouping,
+          'option_group_id' => 'activity_type',
```

Comment:
how you came up with this ? the `setvalue` API call requires both "field" and "value" params to be provided, or have you meant to also change "api.OptionValue.setvalue" to "api.OptionValue.create" here too?

Reply:
if so, then if you do the change it seems the code above will still be an issue, because using "IN" (`'name' => ['IN' => $activityTypes],`) does not seem to work with OptionValue API:

![image](https://user-images.githubusercontent.com/6275540/176900808-e628aeb7-4c27-488f-a78d-8887dc8186d3.png)

Also I don't think the `create` action works with `IN` anyway.